dev-client/App.tsx

```diff
@@ -6,22 +6,33 @@
  */
 
 import React from 'react';
-import {StyleSheet, View} from 'react-native';
+import { StyleSheet, View } from 'react-native';
 
 import Mapbox from '@rnmapbox/maps';
+import { Button, NativeBaseProvider, Text } from 'native-base';
+import { theme } from './theme';
+import LoginView from './components/Login';
+
+
 
 Mapbox.setAccessToken(
   'pk.eyJ1Ijoic2hyb3V4bSIsImEiOiJjbGY4bW8wbGEwbDJnM3FsN3I1ZzBqd2kzIn0.2Alc4o911ooGEtnObLpOUQ',
 );
 
+
 function App(): JSX.Element {
   return (
-    <View style={styles.container}>
-      <Mapbox.MapView style={styles.map} />
-    </View>
+    <NativeBaseProvider theme={theme}>
+      <View style={[styles.container, {
+        justifyContent: "center"
+      }]}>
+        { /* <Mapbox.MapView style={styles.map} /> */}
```

Comment:
Commented this out for now. Will implement some kind of routing as part of this feature and uncomment before we merge into main.